beacon-chain/node/node.go

```diff
@@ -212,6 +212,11 @@ func New(cliCtx *cli.Context, opts ...Option) (*BeaconNode, error) {
 
 	log.Debugln("Starting State Gen")
 	if err := beacon.startStateGen(ctx, bfs, beacon.forkChoicer); err != nil {
+		if errors.Is(err, stategen.ErrNoGenesisBlock) {
+			log.Errorf("No genesis block/state is found. Prysm only provides a mainnet genesis "+
+				"state bundled in the application. You must provide the --%s or --%s flag to load "+
+				"a genesis block/state for this network.", "genesis-state", "genesis-beacon-api-url")
```

Comment:
I see you used variable interopolation for the flag names - did you intend to replace these with direct flag var .Name references?

Reply:
I tried this and got hit with a circular dep. I would have to refactor / move those flags to another package so I just hard coded it in the laziest way. I could remove the templating if that's your feedback here.

Reply:
Makes sense, in that case, yes the templating seems to introduce a smidge of confusion for no discernible benefit, but I don't feel strongly about it.